x-pack/metricbeat/module/prometheus/remote_write/config.go

```diff
@@ -21,6 +25,7 @@ var defaultConfig = config{
 	TypesPatterns: TypesPatterns{
 		CounterPatterns:   nil,
 		HistogramPatterns: nil},
+	Period: time.Second * 60,
 }
 
 func (c *config) Validate() error {
```

Comment:
could this function be a better fit for the `period` validation?

Reply:
I moved this to the Validate() [4cfcf01](https://github.com/elastic/beats/pull/38553/commits/4cfcf016092a21fae39a7dec5bb030222d3f894c)